unity-renderer/Assets/Scripts/CodeStyleExample.cs

```diff
@@ -0,0 +1,130 @@
+﻿// NOTE: following separation lines are used only for clarity and do not take part of our commenting code style!
+//----------------------------------------------------------------------------------------------------  Usings group
+
+using System;
+using System.Collections.Generic;
+using JetBrains.Annotations;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace UIComponents.Scripts.Components.SectionSelector
```

Comment:
i think we should skip including `Scripts` as part of the namespace since it does not describe anything useful

Reply:
Indeed, namespaces get an automatic name by Rider but they are actually their folder structures. 
We should use a Domain approach to names, in this case, a `DCL.UIComponents` is more than enough.